lib/Client.js

```diff
@@ -304,8 +304,10 @@ class Client {
 
   shutdown() {
     TwilioChatClient.shutdown();
+    if (Platform.OS === 'android') {
```

Comment:
For now just did this, because I'm not sure how to create the appropriate one for iOS. The iOS version of the app was not running into the same bug, so it may not be necessary.